CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ You can find its changes [documented below](#060---2020-06-01).
 
 ### Removed
 
+- DPI related functions/methods from `Scale`. ([#1042] by [@xStrom])
+
```

Comment:
And renamed `from_scale` to `new`, `Scale::scale_x` to `Scale::x` , same for `y`.